src/data/restore.ts

```diff
@@ -124,6 +125,7 @@ const restoreElementWithProperties = <
 
 const restoreElement = (
   element: Exclude<ExcalidrawElement, ExcalidrawSelectionElement>,
+  refreshDimensions = true,
```

Comment:
Better to keep this false to avoid breaking changes

Reply:
it's not a breaking change, it's a new default